[2024-10-11T11:41:18.341+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: convert_yolov9_to_trt.convert_to_trt manual__2024-10-11T11:41:16.380408+00:00 [queued]>
[2024-10-11T11:41:18.347+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: convert_yolov9_to_trt.convert_to_trt manual__2024-10-11T11:41:16.380408+00:00 [queued]>
[2024-10-11T11:41:18.347+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-10-11T11:41:18.348+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2024-10-11T11:41:18.348+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-10-11T11:41:18.361+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): convert_to_trt> on 2024-10-11 11:41:16.380408+00:00
[2024-10-11T11:41:18.367+0000] {standard_task_runner.py:55} INFO - Started process 909 to run task
[2024-10-11T11:41:18.370+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'convert_yolov9_to_trt', 'convert_to_trt', 'manual__2024-10-11T11:41:16.380408+00:00', '--job-id', '205', '--raw', '--subdir', 'DAGS_FOLDER/transform/main.py', '--cfg-path', '/tmp/tmp_jvcam9v']
[2024-10-11T11:41:18.370+0000] {standard_task_runner.py:83} INFO - Job 205: Subtask convert_to_trt
[2024-10-11T11:41:18.418+0000] {task_command.py:388} INFO - Running <TaskInstance: convert_yolov9_to_trt.convert_to_trt manual__2024-10-11T11:41:16.380408+00:00 [running]> on host 91eb0b46c706
[2024-10-11T11:41:18.491+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=convert_yolov9_to_trt
AIRFLOW_CTX_TASK_ID=convert_to_trt
AIRFLOW_CTX_EXECUTION_DATE=2024-10-11T11:41:16.380408+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-10-11T11:41:16.380408+00:00
[2024-10-11T11:41:18.492+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-10-11T11:41:18.492+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python3 /opt/***/dags/transform/yolov9/export.py --weights /opt/***/dags/transform/model/model.pt --imgsz 1280 --device 0']
[2024-10-11T11:41:18.511+0000] {subprocess.py:86} INFO - Output:
[2024-10-11T11:41:21.521+0000] {subprocess.py:93} INFO - [34m[1mexport: [0mdata=../../opt/***/dags/transform/yolov9/data/coco.yaml, weights=['/opt/***/dags/transform/model/model.pt'], imgsz=[1280], batch_size=1, device=0, half=False, inplace=False, keras=False, optimize=False, int8=False, dynamic=False, simplify=False, opset=12, verbose=False, workspace=4, nms=False, agnostic_nms=False, topk_per_class=100, topk_all=100, iou_thres=0.45, conf_thres=0.25, include=['engine']
[2024-10-11T11:41:21.539+0000] {subprocess.py:93} INFO - /bin/sh: 1: git: Permission denied
[2024-10-11T11:41:21.593+0000] {subprocess.py:93} INFO - YOLO üöÄ 2024-10-7 Python-3.7.16 torch-1.13.1+cu117 CUDA:0 (NVIDIA GeForce RTX 3070, 7871MiB)
[2024-10-11T11:41:21.593+0000] {subprocess.py:93} INFO - 
[2024-10-11T11:41:22.461+0000] {subprocess.py:93} INFO - Fusing layers...
[2024-10-11T11:41:23.880+0000] {subprocess.py:93} INFO - Model summary: 489 layers, 7105888 parameters, 0 gradients, 26.4 GFLOPs
[2024-10-11T11:41:23.931+0000] {subprocess.py:93} INFO - 
[2024-10-11T11:41:23.931+0000] {subprocess.py:93} INFO - [34m[1mPyTorch:[0m starting from /opt/***/dags/transform/model/model.pt with output shape (1, 84, 33600) (14.4 MB)
[2024-10-11T11:41:24.066+0000] {subprocess.py:93} INFO - [31m[1mrequirements:[0m YOLO requirement "nvidia-tensorrt" not found, attempting AutoUpdate...
[2024-10-11T11:41:43.501+0000] {subprocess.py:93} INFO - 
[2024-10-11T11:41:43.502+0000] {subprocess.py:93} INFO - [notice] A new release of pip available: 22.3.1 -> 24.0
[2024-10-11T11:41:43.502+0000] {subprocess.py:93} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2024-10-11T11:41:43.707+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2024-10-11T11:41:43.707+0000] {subprocess.py:93} INFO - Looking in indexes: https://pypi.ngc.nvidia.com
[2024-10-11T11:41:43.707+0000] {subprocess.py:93} INFO - Collecting nvidia-tensorrt
[2024-10-11T11:41:43.707+0000] {subprocess.py:93} INFO -   Downloading https://developer.download.nvidia.com/compute/redist/nvidia-tensorrt/nvidia_tensorrt-8.4.3.1-cp37-none-linux_x86_64.whl (340.9 MB)
[2024-10-11T11:41:43.707+0000] {subprocess.py:93} INFO -      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 340.9/340.9 MB 4.7 MB/s eta 0:00:00
[2024-10-11T11:41:43.707+0000] {subprocess.py:93} INFO - Requirement already satisfied: nvidia-cublas-cu11 in /home/***/.local/lib/python3.7/site-packages (from nvidia-tensorrt) (11.10.3.66)
[2024-10-11T11:41:43.707+0000] {subprocess.py:93} INFO - Requirement already satisfied: nvidia-cuda-runtime-cu11 in /home/***/.local/lib/python3.7/site-packages (from nvidia-tensorrt) (11.7.99)
[2024-10-11T11:41:43.707+0000] {subprocess.py:93} INFO - Requirement already satisfied: nvidia-cudnn-cu11 in /home/***/.local/lib/python3.7/site-packages (from nvidia-tensorrt) (8.5.0.96)
[2024-10-11T11:41:43.707+0000] {subprocess.py:93} INFO - Requirement already satisfied: setuptools in /usr/local/lib/python3.7/site-packages (from nvidia-cublas-cu11->nvidia-tensorrt) (57.5.0)
[2024-10-11T11:41:43.708+0000] {subprocess.py:93} INFO - Requirement already satisfied: wheel in /usr/local/lib/python3.7/site-packages (from nvidia-cublas-cu11->nvidia-tensorrt) (0.38.4)
[2024-10-11T11:41:43.708+0000] {subprocess.py:93} INFO - Installing collected packages: nvidia-tensorrt
[2024-10-11T11:41:43.708+0000] {subprocess.py:93} INFO - Successfully installed nvidia-tensorrt-8.4.3.1
[2024-10-11T11:41:43.708+0000] {subprocess.py:93} INFO - 
[2024-10-11T11:41:43.708+0000] {subprocess.py:93} INFO - [31m[1mrequirements:[0m 1 package updated per ['nvidia-tensorrt']
[2024-10-11T11:41:43.708+0000] {subprocess.py:93} INFO - [31m[1mrequirements:[0m ‚ö†Ô∏è [1mRestart runtime or rerun command for updates to take effect[0m
[2024-10-11T11:41:43.708+0000] {subprocess.py:93} INFO - 
[2024-10-11T11:41:43.803+0000] {subprocess.py:93} INFO - [31m[1mrequirements:[0m YOLO requirement "onnx" not found, attempting AutoUpdate...
[2024-10-11T11:41:51.788+0000] {subprocess.py:93} INFO - ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
[2024-10-11T11:41:51.789+0000] {subprocess.py:93} INFO - tensorboard 2.11.2 requires protobuf<4,>=3.9.2, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.789+0000] {subprocess.py:93} INFO - mysql-connector-python 8.0.32 requires protobuf<=3.20.3,>=3.11.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.789+0000] {subprocess.py:93} INFO - google-cloud-workflows 1.7.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.789+0000] {subprocess.py:93} INFO - google-cloud-vision 1.0.2 requires protobuf<4.0.0dev, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.789+0000] {subprocess.py:93} INFO - google-cloud-videointelligence 1.16.3 requires protobuf<4.0.0dev, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.789+0000] {subprocess.py:93} INFO - google-cloud-translate 1.7.2 requires protobuf<4.0.0dev, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.789+0000] {subprocess.py:93} INFO - google-cloud-texttospeech 1.0.3 requires protobuf<4.0.0dev, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.789+0000] {subprocess.py:93} INFO - google-cloud-tasks 2.10.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.789+0000] {subprocess.py:93} INFO - google-cloud-speech 1.3.4 requires protobuf<4.0.0dev, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.789+0000] {subprocess.py:93} INFO - google-cloud-spanner 1.19.3 requires protobuf<4.0.0dev, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.789+0000] {subprocess.py:93} INFO - google-cloud-secret-manager 1.0.2 requires protobuf<4.0.0dev, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.789+0000] {subprocess.py:93} INFO - google-cloud-resource-manager 1.6.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-redis 2.9.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-pubsub 2.13.5 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-os-login 2.7.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-orchestration-*** 1.4.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-monitoring 2.11.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-memcache 1.4.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-logging 3.2.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-language 1.3.2 requires protobuf<4.0.0dev, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-kms 2.12.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-dlp 1.0.2 requires protobuf<4.0.0dev, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-dataproc 5.0.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-dataproc-metastore 1.6.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-dataplex 1.1.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-dataform 0.2.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-datacatalog 3.9.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-container 2.11.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-build 3.9.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-bigtable 1.7.3 requires protobuf<4.0.0dev, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.790+0000] {subprocess.py:93} INFO - google-cloud-bigquery 2.34.4 requires protobuf<4.0.0dev,>=3.12.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.791+0000] {subprocess.py:93} INFO - google-cloud-bigquery-storage 2.14.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.791+0000] {subprocess.py:93} INFO - google-cloud-bigquery-datatransfer 3.7.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.791+0000] {subprocess.py:93} INFO - google-cloud-automl 2.8.0 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.791+0000] {subprocess.py:93} INFO - google-cloud-appengine-logging 1.1.3 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.791+0000] {subprocess.py:93} INFO - google-cloud-aiplatform 1.16.1 requires protobuf<4.0.0dev,>=3.19.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.791+0000] {subprocess.py:93} INFO - google-ads 18.0.0 requires protobuf!=3.18.*,!=3.19.*,<=3.20.0,>=3.12.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.791+0000] {subprocess.py:93} INFO - apache-***-providers-google 8.8.0 requires protobuf<=3.20.0, but you have protobuf 4.24.4 which is incompatible.
[2024-10-11T11:41:51.795+0000] {subprocess.py:93} INFO - 
[2024-10-11T11:41:51.795+0000] {subprocess.py:93} INFO - [notice] A new release of pip available: 22.3.1 -> 24.0
[2024-10-11T11:41:51.795+0000] {subprocess.py:93} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2024-10-11T11:41:51.958+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2024-10-11T11:41:51.958+0000] {subprocess.py:93} INFO - Collecting onnx
[2024-10-11T11:41:51.958+0000] {subprocess.py:93} INFO -   Downloading onnx-1.14.1-cp37-cp37m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (14.6 MB)
[2024-10-11T11:41:51.958+0000] {subprocess.py:93} INFO -      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 14.6/14.6 MB 26.6 MB/s eta 0:00:00
[2024-10-11T11:41:51.959+0000] {subprocess.py:93} INFO - Requirement already satisfied: numpy in /home/***/.local/lib/python3.7/site-packages (from onnx) (1.21.1)
[2024-10-11T11:41:51.959+0000] {subprocess.py:93} INFO - Collecting protobuf>=3.20.2
[2024-10-11T11:41:51.959+0000] {subprocess.py:93} INFO -   Downloading protobuf-4.24.4-cp37-abi3-manylinux2014_x86_64.whl (311 kB)
[2024-10-11T11:41:51.959+0000] {subprocess.py:93} INFO -      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 311.6/311.6 kB 77.6 MB/s eta 0:00:00
[2024-10-11T11:41:51.959+0000] {subprocess.py:93} INFO - Requirement already satisfied: typing-extensions>=3.6.2.1 in /home/***/.local/lib/python3.7/site-packages (from onnx) (4.4.0)
[2024-10-11T11:41:51.959+0000] {subprocess.py:93} INFO - Installing collected packages: protobuf, onnx
[2024-10-11T11:41:51.959+0000] {subprocess.py:93} INFO -   Attempting uninstall: protobuf
[2024-10-11T11:41:51.959+0000] {subprocess.py:93} INFO -     Found existing installation: protobuf 3.20.0
[2024-10-11T11:41:51.959+0000] {subprocess.py:93} INFO -     Uninstalling protobuf-3.20.0:
[2024-10-11T11:41:51.959+0000] {subprocess.py:93} INFO -       Successfully uninstalled protobuf-3.20.0
[2024-10-11T11:41:51.959+0000] {subprocess.py:93} INFO - Successfully installed onnx-1.14.1 protobuf-4.24.4
[2024-10-11T11:41:51.959+0000] {subprocess.py:93} INFO - 
[2024-10-11T11:41:51.959+0000] {subprocess.py:93} INFO - [31m[1mrequirements:[0m 1 package updated per ['onnx']
[2024-10-11T11:41:51.959+0000] {subprocess.py:93} INFO - [31m[1mrequirements:[0m ‚ö†Ô∏è [1mRestart runtime or rerun command for updates to take effect[0m
[2024-10-11T11:41:51.959+0000] {subprocess.py:93} INFO - 
[2024-10-11T11:41:51.982+0000] {subprocess.py:93} INFO - 
[2024-10-11T11:41:51.982+0000] {subprocess.py:93} INFO - [34m[1mONNX:[0m starting export with onnx 1.14.1...
[2024-10-11T11:41:53.485+0000] {subprocess.py:93} INFO - [34m[1mONNX:[0m export success ‚úÖ 9.7s, saved as /opt/***/dags/transform/model/model.onnx (27.9 MB)
[2024-10-11T11:41:53.486+0000] {subprocess.py:93} INFO - 
[2024-10-11T11:41:53.486+0000] {subprocess.py:93} INFO - [34m[1mTensorRT:[0m starting export with TensorRT 8.4.3.1...
[2024-10-11T11:41:53.781+0000] {subprocess.py:93} INFO - [10/11/2024-11:41:53] [TRT] [I] [MemUsageChange] Init CUDA: CPU +571, GPU +0, now: CPU 3711, GPU 963 (MiB)
[2024-10-11T11:41:54.597+0000] {subprocess.py:93} INFO - [10/11/2024-11:41:54] [TRT] [I] [MemUsageChange] Init builder kernel library: CPU +437, GPU +104, now: CPU 4167, GPU 1067 (MiB)
[2024-10-11T11:41:54.620+0000] {subprocess.py:93} INFO - [10/11/2024-11:41:54] [TRT] [I] ----------------------------------------------------------------
[2024-10-11T11:41:54.620+0000] {subprocess.py:93} INFO - [10/11/2024-11:41:54] [TRT] [I] Input filename:   /opt/***/dags/transform/model/model.onnx
[2024-10-11T11:41:54.620+0000] {subprocess.py:93} INFO - [10/11/2024-11:41:54] [TRT] [I] ONNX IR version:  0.0.7
[2024-10-11T11:41:54.620+0000] {subprocess.py:93} INFO - [10/11/2024-11:41:54] [TRT] [I] Opset version:    12
[2024-10-11T11:41:54.621+0000] {subprocess.py:93} INFO - [10/11/2024-11:41:54] [TRT] [I] Producer name:    pytorch
[2024-10-11T11:41:54.621+0000] {subprocess.py:93} INFO - [10/11/2024-11:41:54] [TRT] [I] Producer version: 1.13.1
[2024-10-11T11:41:54.621+0000] {subprocess.py:93} INFO - [10/11/2024-11:41:54] [TRT] [I] Domain:
[2024-10-11T11:41:54.621+0000] {subprocess.py:93} INFO - [10/11/2024-11:41:54] [TRT] [I] Model version:    0
[2024-10-11T11:41:54.621+0000] {subprocess.py:93} INFO - [10/11/2024-11:41:54] [TRT] [I] Doc string:
[2024-10-11T11:41:54.621+0000] {subprocess.py:93} INFO - [10/11/2024-11:41:54] [TRT] [I] ----------------------------------------------------------------
[2024-10-11T11:41:54.653+0000] {subprocess.py:93} INFO - [10/11/2024-11:41:54] [TRT] [W] onnx2trt_utils.cpp:369: Your ONNX model has been generated with INT64 weights, while TensorRT does not natively support INT64. Attempting to cast down to INT32.
[2024-10-11T11:41:54.977+0000] {subprocess.py:93} INFO - /opt/***/dags/transform/yolov9/export.py:287: DeprecationWarning: Use set_memory_pool_limit instead.
[2024-10-11T11:41:54.978+0000] {subprocess.py:93} INFO -   config.max_workspace_size = workspace * 1 << 30
[2024-10-11T11:41:54.978+0000] {subprocess.py:93} INFO - [34m[1mTensorRT:[0m input "images" with shape(1, 3, 1280, 1280) DataType.FLOAT
[2024-10-11T11:41:54.978+0000] {subprocess.py:93} INFO - [34m[1mTensorRT:[0m output "output0" with shape(1, 84, 33600) DataType.FLOAT
[2024-10-11T11:41:54.978+0000] {subprocess.py:93} INFO - [34m[1mTensorRT:[0m building FP32 engine as /opt/***/dags/transform/model/model.engine
[2024-10-11T11:42:01.854+0000] {subprocess.py:93} INFO - [10/11/2024-11:42:01] [TRT] [I] [MemUsageChange] Init cuBLAS/cuBLASLt: CPU +0, GPU +8, now: CPU 4204, GPU 1075 (MiB)
[2024-10-11T11:42:01.856+0000] {subprocess.py:93} INFO - [10/11/2024-11:42:01] [TRT] [I] [MemUsageChange] Init cuDNN: CPU +0, GPU +10, now: CPU 4204, GPU 1085 (MiB)
[2024-10-11T11:42:01.860+0000] {subprocess.py:93} INFO - [10/11/2024-11:42:01] [TRT] [I] Local timing cache in use. Profiling results in this builder pass will not be stored.
[2024-10-11T11:42:58.958+0000] {subprocess.py:93} INFO - [10/11/2024-11:42:58] [TRT] [I] Some tactics do not have sufficient workspace memory to run. Increasing workspace size will enable more tactics, please check verbose output for requested sizes.
[2024-10-11T11:43:34.977+0000] {subprocess.py:93} INFO - [10/11/2024-11:43:34] [TRT] [I] Detected 1 inputs and 3 output network tensors.
[2024-10-11T11:43:35.430+0000] {subprocess.py:93} INFO - [10/11/2024-11:43:35] [TRT] [I] Total Host Persistent Memory: 248960
[2024-10-11T11:43:35.430+0000] {subprocess.py:93} INFO - [10/11/2024-11:43:35] [TRT] [I] Total Device Persistent Memory: 5580800
[2024-10-11T11:43:35.430+0000] {subprocess.py:93} INFO - [10/11/2024-11:43:35] [TRT] [I] Total Scratch Memory: 10125824
[2024-10-11T11:43:35.430+0000] {subprocess.py:93} INFO - [10/11/2024-11:43:35] [TRT] [I] [MemUsageStats] Peak memory usage of TRT CPU/GPU memory allocators: CPU 0 MiB, GPU 0 MiB
[2024-10-11T11:43:35.580+0000] {subprocess.py:93} INFO - [10/11/2024-11:43:35] [TRT] [I] [BlockAssignment] Algorithm ShiftNTopDown took 149.48ms to assign 9 blocks to 371 nodes requiring 140083200 bytes.
[2024-10-11T11:43:35.580+0000] {subprocess.py:93} INFO - [10/11/2024-11:43:35] [TRT] [I] Total Activation Memory: 140083200
[2024-10-11T11:43:35.596+0000] {subprocess.py:93} INFO - [10/11/2024-11:43:35] [TRT] [I] [MemUsageChange] Init cuDNN: CPU +0, GPU +10, now: CPU 4246, GPU 1145 (MiB)
[2024-10-11T11:43:35.601+0000] {subprocess.py:93} INFO - [10/11/2024-11:43:35] [TRT] [I] [MemUsageChange] TensorRT-managed allocation in building engine: CPU +0, GPU +0, now: CPU 0, GPU 0 (MiB)
[2024-10-11T11:43:35.602+0000] {subprocess.py:93} INFO - [10/11/2024-11:43:35] [TRT] [W] The getMaxBatchSize() function should not be used with an engine built from a network created with NetworkDefinitionCreationFlag::kEXPLICIT_BATCH flag. This function will always return 1.
[2024-10-11T11:43:35.942+0000] {subprocess.py:93} INFO - [10/11/2024-11:43:35] [TRT] [W] The getMaxBatchSize() function should not be used with an engine built from a network created with NetworkDefinitionCreationFlag::kEXPLICIT_BATCH flag. This function will always return 1.
[2024-10-11T11:43:36.342+0000] {subprocess.py:93} INFO - /opt/***/dags/transform/yolov9/export.py:314: DeprecationWarning: Use build_serialized_network instead.
[2024-10-11T11:43:36.342+0000] {subprocess.py:93} INFO -   with builder.build_engine(network, config) as engine, open(f, 'wb') as t:
[2024-10-11T11:43:36.342+0000] {subprocess.py:93} INFO - [34m[1mTensorRT:[0m export success ‚úÖ 132.4s, saved as /opt/***/dags/transform/model/model.engine (47.2 MB)
[2024-10-11T11:43:36.342+0000] {subprocess.py:93} INFO - 
[2024-10-11T11:43:36.342+0000] {subprocess.py:93} INFO - Export complete (134.8s)
[2024-10-11T11:43:36.342+0000] {subprocess.py:93} INFO - Results saved to [1m/opt/***/dags/transform/model[0m
[2024-10-11T11:43:36.342+0000] {subprocess.py:93} INFO - Detect:          python detect.py --weights /opt/***/dags/transform/model/model.engine
[2024-10-11T11:43:36.343+0000] {subprocess.py:93} INFO - Validate:        python val.py --weights /opt/***/dags/transform/model/model.engine
[2024-10-11T11:43:36.343+0000] {subprocess.py:93} INFO - PyTorch Hub:     model = torch.hub.load('ultralytics/yolov5', 'custom', '/opt/***/dags/transform/model/model.engine')
[2024-10-11T11:43:36.343+0000] {subprocess.py:93} INFO - Visualize:       https://netron.app
[2024-10-11T11:43:37.387+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-11T11:43:37.408+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=convert_yolov9_to_trt, task_id=convert_to_trt, execution_date=20241011T114116, start_date=20241011T114118, end_date=20241011T114337
[2024-10-11T11:43:37.448+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-10-11T11:43:37.461+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
